src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -56,5 +56,4 @@ dataverse.oai.server.maxsets=100
 #dataverse.oai.server.repositoryname=
 
 # AUTHENTICATION
-dataverse.auth.oidc.pkce.max-cache-size=10000
-dataverse.auth.oidc.pkce.max-cache-age=300
+payara.security.openid.sessionScopedConfiguration=true
```

Comment:
What about non-Docker installation of Dataverse? Is payara.security.openid.sessionScopedConfiguration=true being configured?

Reply:
It's within the included config file so it will be used by all installations. As this ships with the WAR, this is not at all related to Docker or No Docker.

Reply:
It is the multi-tenant part: https://docs.payara.fish/enterprise/docs/Technical%20Documentation/Public%20API/OpenID%20Connect%20Support.html